worker/caasunitprovisioner/application_worker.go

```diff
@@ -189,6 +202,20 @@ func (aw *applicationWorker) loop() error {
 					return errors.Trace(err)
 				}
 			}
+		case _, ok := <-appOperatorWatcher.Changes():
+			if !ok {
+				return errors.New("operator watcher channel closed")
+			}
+			logger.Debugf("operator update for %v", aw.application)
+			operator, err := aw.containerBroker.Operator(aw.application)
+			if err != nil {
+				return errors.Trace(err)
```

Comment:
We should probably check for a not found error here. If not found, the best thing to do I think is set the status as Stopped/Terminated